src/scripts/parser.js

```diff
@@ -20,7 +20,16 @@ exports.deflate = function (json, prefix) {
             result = result.concat(exports.deflate(json[ key ], _prefix));
         } else {
             _prefix = prefix ? prefix.concat(key) : key;
-            result.push(_prefix.concat('=').concat(json[ key ]));
+            var value = json[key];
```

Comment:
Before adding the value, check if any of these that need special treatment, if so, put quotes around.